packages/ynap-parsers/src/bank2ynab/bank2ynab.ts

```diff
@@ -65,7 +65,13 @@ export const generateParser = (config: ParserConfig) => {
     {} as { [k in keyof YnabRow]: number },
   );
 
-  const hasCol = (name: keyof typeof columns) => Object.keys(columns).includes(name);
+  const hasCol = (name: keyof typeof columns) => {
+    if (typeof name !== 'string') {
```

Comment:
Could you explain in which scenario this check is necessary? Did you run into a bug because of it? :)

Reply:
This is weird. I was getting tslint error saying that name can have multiple types. Maybe after yarn install, it has updated. Anyway, I've reverted this check.